modules/admin_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -29,16 +29,22 @@ You have control of a number of user permissions on file shares:
 ** Allow users to send mail notification for shared files
 ** Set the language used for public mail notification for shared files
 ** Allow users to share file via social media
+* Set default expiration date for user shares
+** Set the number of days to expire after
+** Enforce as maximum expiration date
+* Set default expiration date for group shares
+** Set the number of days to expire after
+** Enforce as maximum expiration date
 * Automatically accept new incoming local user shares
 * Allow resharing
-* Default user and group share permissions
+* Allow sharing with groups
 ** Restrict users to only share with users in their groups
```

Comment:
Not sure why the two restrictions are on level 2 (`**`). Actually they are independent options on the first level.

Reply:
You are correct - fixed.